concatenateHDF5.m

```diff
@@ -0,0 +1,50 @@
+function [hdf5Name] = concatenateHDF5(tifDir)
+% Concatenates all tif files in the specified directory into an
+% hdf5 file. A file structure similar to the one below should be used. The
+% stored files are all space binned by the factor set in 'downsmpFactor'.
+%
+% Example file structure: "G:\cohort9-herrin224\mouse5\day10_allo-south"
+% Example use: "concatenateHDF5(tifDir)" where 'tifDir' points to the
+%   folder containing the tif files
+%
+% 2015-01-06 Jessica Maxey
+
+cd(tifDir);
+list = ls('*.tif');
+firstName = strtrim(list(1,:));
+[rows,cols] = size(imread(firstName));
+numFiles = size(list,1);
+total_frames = count_frames_in_tif(tifDir);
+
+totalFrames = 1;
+[drive,buffer] = strtok(tifDir,'\');
+[cohort,buffer] = strtok(buffer,'\');
+[mouse,buffer] = strtok(buffer,'\');
+[day,buffer] = strtok(buffer,'\');
+
```

Comment:
Just a comment: If you allow the user to specify the filename (as a function parameter) rather than constructing it from the path, you'd relax the requirement on the path structure.